index.html

```diff
@@ -4939,18 +4939,56 @@ <h3>Unsigned Claims</h3>
       </section>
 
       <section class="informative">
-        <h3>Token Binding</h3>
+        <h3>Man-in-the-Middle (MITM), Replay, and Cloning Attacks</h3>
+
+        <p>
+The data model does not inherently prevent
+<a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">Man-in-the-Middle (MITM)</a>,
+<a href="https://en.wikipedia.org/wiki/Replay_attack">replay</a>, and 
+<a href="https://en.wikipedia.org/wiki/Spoofing_attack">spoofing</a> attacks.
+Both online and offline use cases may be susceptible to these types of attacks,
+where an adversary intercepts, modifies, re-uses, and/or replicates the verifiable
+credential data during transmission or storage. 
+        </p>
+        <h4>Man-in-the-Middle (MITM) Attack</h4>
 
         <p>
 A <a>verifier</a> might need to ensure it is the intended recipient of a
 <a>verifiable presentation</a> and not the target of a
 <a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">man-in-the-middle attack</a>.
-Approaches such as token binding [[RFC8471]], which ties the request for a
-<a>verifiable presentation</a> to the response, can secure the protocol.
+Some <a href=#securing-verifiable-credentials>securing mechanisms</a>, like [[VC-JOSE-COSE]],
+provide an option to specify the intended audience of a presentation, which can
+help reduce this risk.
+        </p>
+        <p>
+Alternate approaches such as token binding [[RFC8471]], which ties the request
+for a <a>verifiable presentation</a> to the response, can secure the protocol.
 Any unsecured protocol is susceptible to man-in-the-middle attacks.
         </p>
+        <h4>Replay Attack</h4>
+
+        <p>
+A <a>verifier</a> may wish to ensure that a presentation is not used more than a
+certain number of times. For example, a credential representing an event ticket,
+may allow entry to multiple individuals if presented multiple times, undermining
+the utility of the ticket. To prevent against such attacks verifiers can make use
+of techniques such as requiring the inclusion of a 
+<a href="https://en.wikipedia.org/wiki/Cryptographic_nonce">nonce</a> during presentation,
+or add an expiry timestamp to reduce the window of attack.
+        </p> 
+        <h4>Spoofing Attack</h4>
+
+        <p>
+A <a>verifier</a> has a vested interest in knowing that a <a>holder</a> is authorized
+to present the claims inside of a <a>verifiable presentation</a>. While the data
+model outlines the structure and data elements necessary for a credential, it does
+not inherently include a foolproof mechanism to ascertain the authorization of
+presented credentials. To address this concern, implementers may need to explore
+supplementary methods, such as binding verifiable credentials to strong
+authentication mechanisms or using additional attributes in presentations
+to enable legitimate proof of control.
```

Comment:
```suggestion
to enable proof of control.
```

It's not clear what "legitimate" would mean.

Reply:
Right, but remember, "legitimate" proof of control isn't always up to the issuer and can be use case dependent. If I am fleeing a despotic regime, but need to provide documentation of that fact to the place to which I am fleeing, the despotic regime might not approve of my usage of the credential as "legitimate", but the country to which I'm fleeing would see it as legitimate.

I'm saying that adding this word make the sentence dicey, while removing it makes the sentence more palatable.

Reply:
good point. I have accepted your suggestion